packages/teleport-cli/src/constants.ts

```diff
@@ -0,0 +1,23 @@
+export const CONFIG_FILE_NAME = `teleport.config`
+export const CONIFG_FILE_EXTENSION = '.json'
+export const DEFAULT_CONFIG_FILE_NAME = `${CONFIG_FILE_NAME}${CONIFG_FILE_EXTENSION}`
+
+export const DEFALT_CONFIG_TEMPLATE: DefaultConfigTemplate = {
+  project: {},
+  components: {},
+}
+
+export const BASE_URL = 'https://us-central1-croapp-dev.cloudfunctions.net/repl-api/'
+export const STUDIO_URL = 'https://playground-api-production-v5.services.teleporthq.io/project/'
```

Comment:
Shouldn't this URL be the frontend one?

Reply:
If not, it's a little tricky, because we might upgrade the backend in the future, let's say from v5 to v6.

Reply:
So in that case, an old version of the CLI will not work anymore